src/standalone/Vegetation/Canopy.jl

```diff
@@ -421,12 +422,23 @@ function ClimaLSM.make_update_aux(
         T::FT = canopy.atmos.T(t)
         h::FT = canopy.atmos.h
         q::FT = canopy.atmos.q(t)
+        ref_time = canopy.atmos.ref_time
 
         # update radiative transfer
         RT = canopy.radiative_transfer
         K = extinction_coeff(ld, θs)
         PAR .= compute_PAR(RT, canopy.radiation, t)
         NIR .= compute_NIR(RT, canopy.radiation, t)
+        e_sat =
+            Thermodynamics.saturation_vapor_pressure.(
+                Ref(thermo_params),
+                T,
+                Ref(Thermodynamics.Liquid()),
+            )
+        e = FT(2.655 * q * P / (q + 1.6455))
```

Comment:
Ill figure this out, but we should use a Thermodynamics.jl function for this: https://github.com/CliMA/Thermodynamics.jl/blob/e7a012f7ddc883fa307f5a2e672304ef3b49285b/src/relations.jl#L122 maybe?

Reply:
I think this should work:
`e = Thermodynamics.partial_pressure_vapor.(Ref(thermo_params), P, Ref(PhasePartition(q))`

Reply:
Great! Got it.